pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentUploadRestletResource.java

```diff
@@ -303,35 +303,37 @@ private SuccessResponse uploadSegment(FormDataMultiPart multiPart, boolean enabl
           throw new UnsupportedOperationException("Unsupported upload type: " + uploadType);
       }
 
+      String rawTableName = segmentMetadata.getTableName();
```

Comment:
final?

Reply:
All other temporary variables in the same code block is not using `final`. Do you think that we still need `final`?